src/reconstruction/refinement/GPRLogic/Node.m

```diff
@@ -131,41 +131,96 @@ function addChild(self,childNode)
         end
         
         function [geneSets,genePos] = getFunctionalGeneSets(self,geneNames)
-        % Get all functional gene sets useable for this node
-        % USAGE:
-        %    geneSets = Node.getFunctionalGeneSets(geneNames)                
-        %    
-        % INPUTS:
-        %    geneNames:   the genes in the order represented by the
-        %                 literals (which represent positions)         
-        %
-        % OUTPUTS:
-        %    geneSets:    A cell array of gene Combinations that would make this node active
-        %
-        %    genePos:     A set of positions (according to the parsed
-        %                 Rules) of genes that would make this tree
-        %                 evaluate to true.
-        % 
-            dnfNode = self.convertToDNF();                                        
+            % Get all functional gene sets useable for this node
```

Comment:
Any reason for these indents here?

Reply:
err no. I'll correct this

Reply:
Actually: This is the default indentation matlab does.
So maybe keep it, as any `ctrl+A / ctrl-I` call will reintroduce it?